srcpkgs/NetworkManager/patches/fix-cross-gir.patch

```diff
@@ -0,0 +1,29 @@
+diff --git a/libnm/meson.build b/libnm/meson.build
+index 558a587..902bfe0 100644
+--- libnm/meson.build
++++ libnm/meson.build
+@@ -244,7 +244,23 @@ if enable_introspection
+     )
+   endif
+ 
+-  generate_setting_docs = join_paths(meson.current_source_dir(), 'generate-setting-docs.py')
++  if meson.is_cross_build()
++      arch = host_machine.cpu_family()
++      if arch == 'x86'
++              qemu_arch = 'ia32'
```

Comment:
There is no ia32, you are looking for i386

Reply:
Oops, copied that from fwupd